appengine/flexible/tasks/app.flexible.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+runtime: python
+env: flex
+entrypoint: gunicorn -b :$PORT main:app
```

Comment:
Can we change this to "gunicorn -b :$PORT --threads=4 main:app" to ensure the user does not run into concurrency issues while dealing with asynchronously triggered tasks?